testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/OfflineTokenTest.java

```diff
@@ -1050,10 +1050,6 @@ private void offlineTokenRequestWithScopeParameter(String expectedRefreshAlg, St
 
         Assert.assertEquals(TokenUtil.TOKEN_TYPE_OFFLINE, offlineToken.getType());
         Assert.assertEquals(0, offlineToken.getExpiration());
-
-        //refresh token without sending offline_access scope
```

Comment:
Why you are removing this check? The `offline_access` scope should be there as it was request in the first token request, no?

Reply:
> Why you are removing this check? The `offline_access` scope should be there as it was request in the first token request, no?

This test has been changed in previous [false accepted PR ](https://github.com/keycloak/keycloak/pull/28093).
Refresh flow is offline access and scope parameter is '_phone_'. So, access token must not contain _offline_access_ and refresh access token must contain _offline_access_.

So, we need to revert back changes in _testRefreshWithOfflineToken_ in order to take into account if scope parameter will contain _offline_access_.
However, this test part is same as test method _offlineTokenRefreshWithoutOfflineAccessScope_. So, I have enchanced method _offlineTokenRefreshWithoutOfflineAccessScope_. I believe it is enough.